modules/virt-customizing-storage-profile.adoc

```diff
@@ -67,3 +67,42 @@ status:
 <2> The `volumeMode` that you select.
 +
 After you save your changes, the selected values appear in the storage profile `status` element.
+
+== Setting a default cloning strategy using a storage profile
+
+You can use storage profiles to set a default cloning method for a storage class, creating a _cloning strategy_. Setting cloning strategies can be helpful, for example, if your storage vendor only supports certain cloning methods. It also allows you to select a method that limits resource usage or maximizes performance.
+
+Cloning strategies can be specified by setting the `cloneStrategy` attribute in a storage profile to one of these values:
+
+* `snapshot` - This method will be used by default when snapshots are configured. This cloning strategy uses a temporary volume snapshot to clone the volume. The storage provisioner must support CSI snapshots.
+* `copy` - This method uses a source Pod and a target Pod to copy data from the source volume to the target volume. Host-assisted cloning is the least efficient method of cloning.
+* `csi-clone` - This method uses the CSI clone API to efficiently clone an existing volume without using an interim volume snapshot. Unlike `snapshot` or `copy`, which are used by default if no storage profile is defined, CSI volume cloning is only used when you specify it in the `StorageProfile` object for the provisioner's storage class.
++
+[NOTE]
+====
+You can also set clone strategies usng the CLI without modifying the default `claimPropertySets` in your YAML `spec`.
+====
++
+.Example storage profile
+[source,yaml]
+----
+apiVersion: cdi.kubevirt.io/v1beta1
+kind: StorageProfile
+metadata:
+  name: <provisioner_class>
+#   ...
+spec:
+  claimPropertySets:
+  - accessModes:
+    - ReadWriteOnce <1>
+    volumeMode:
+      Filesystem <2>
+  cloneStrategy:
+      csi-clone <3>
```

Comment:
csi-clone - remove two spaces from indentation, it is directly under cloneStrategy

Reply:
Thanks. fixed it.